### PR TITLE
[2.11.x] DDF-3556 Updates default value for STS issuer to IdP login URI.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Updates the STS issuer to align with the SAML spec:

[SAML Profiles, Section 4.1.4.2](https://www.oasis-open.org/committees/download.php/56782/sstc-saml-profiles-errata-2.0-wd-07.pdf)

> Each assertion's `<Issuer>` element MUST contain the unique identifier of the [E26]responding identity provider; the `Format` attribute MUST be omitted or have a value of `urn:oasis:names:tc:SAML:2.0:nameid-format:entity`. Note that this profile assumes a single responding identity provider, and all assertions in a response MUST be issued by the same entity.

#### Who is reviewing it? 
@blen-desta @ricklarsen 

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
@codice/security 

#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@stustison

#### How should this be tested? (List steps with links to updated documentation)
- Full CI build.
- Interop with Spring SP and confirm that the POST binding works correctly.

#### What are the relevant tickets?
[DDF-3556](https://codice.atlassian.net/browse/DDF-3556)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/cxf-sts.xml [hunk 1]
```diff
@@ -258,7 +258,7 @@
         <argument ref="STSProperties"/>
         <cm:managed-properties persistent-id="ddf.security.sts"
                                update-strategy="container-managed"/>
-        <property name="issuer" value="${org.codice.ddf.system.hostname}"/>
+        <property name="issuer" value="https://${org.codice.ddf.system.hostname}:${org.codice.ddf.system.httpsPort}/services/idp/login"/>
```
Comment (clockard) [inline platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/cxf-sts.xml]: If you want `services` could also be replaced with `${org.codice.ddf.system.rootContext}` if you wanted.

Comment (stustison) [reply platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/cxf-sts.xml]: Yeah it is set via a config, so probably a good thing to do.

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

(Nexus issue)

**Pull request merged**